adev/shared-docs/pipeline/api-gen/rendering/templates/cli-card.tsx

```diff
@@ -31,8 +31,18 @@ export function CliCard(props: {card: CliCardRenderable}) {
               <div dangerouslySetInnerHTML={{__html: item.description}}></div>
             </div>
             <div class="docs-reference-type-and-default">
-              <span>Value Type</span>
-              <code>{item.type}</code>
+              {/* Display the enum values if there are some, else the type expected for the option */}
+              {item.enum ? (
+                <>
+                  <span>Enum Values</span>
```

Comment:
I think in a MAN page Allowed Values is a better fit.

```suggestion
                  <span>Allowed Values</span>
```